### PR TITLE
media-sound/yoshimi: fix #729716

### PR DESCRIPTION
**Pull request by RarogCmex**
Created new ebuild -r1 with patch from bugzilla.
Fixed dependencies
Closes:https://bugs.gentoo.org/729716

Package-Manager: Portage-2.3.99, Repoman-2.3.23
RepoMan-Options: --force
Signed-off-by: Denis Reva <denis7774@gmail.com>

### CODE REVIEW

File: media-sound/yoshimi/yoshimi-1.7.1.ebuild [hunk 1]
```diff
@@ -27,7 +27,7 @@ DEPEND="
 	virtual/jack
 	x11-libs/cairo[X]
 	x11-libs/fltk:1[opengl]
-	lv2? ( media-libs/lv2 )
+	lv2? ( <media-libs/lv2-1.18.0 )
```
Comment (juippis) [inline media-sound/yoshimi/yoshimi-1.7.1.ebuild:30]: Looks good, but this needs a revbump. Ideally it'd be in a separate commit, too. You can use `git mv` here.

**Comment by RarogCmex** [comment]
I've force-pushed new version of ebuild, with ~ keyword because it's potentially unstable, and as I know proxy maintainers should no keyword objects.

File: media-sound/yoshimi/yoshimi-1.7.1.ebuild (hunk 1 above)
Comment (RarogCmex) [reply media-sound/yoshimi/yoshimi-1.7.1.ebuild]: Thanks, I don't know that media-libs/lv2 is updated.

File: media-sound/yoshimi/yoshimi-1.7.1-r1.ebuild [hunk 2]
```diff
@@ -11,7 +11,7 @@ SRC_URI="https://github.com/${PN^}/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
 
 LICENSE="GPL-2"
 SLOT="0"
-KEYWORDS="amd64 x86"
+KEYWORDS="~amd64 ~x86"
```
Comment (juippis) [inline media-sound/yoshimi/yoshimi-1.7.1-r1.ebuild:14]: If you edit KEYWORDS then do not revbump, we need to leave a stable version in tree.

**Comment by RarogCmex** [comment]
@juippis Fixed.

File: media-sound/yoshimi/yoshimi-1.7.1-r1.ebuild (hunk 2 above)
Comment (RarogCmex) [reply media-sound/yoshimi/yoshimi-1.7.1-r1.ebuild]: Thanks

File: media-sound/yoshimi/yoshimi-1.7.1-r1.ebuild [hunk 3]
```diff
@@ -35,6 +35,8 @@ CMAKE_USE_DIR="${WORKDIR}/${P}/src"
 
 DOCS=( Changelog README.txt )
 
+PATCHES=( ${FILESDIR}/yoshimi-1.7.1-fix-LV2_Descriptor.patch )
```
Comment (thesamesam) [inline media-sound/yoshimi/yoshimi-1.7.1-r1.ebuild:38]: I think we need quoting on at least "${FILESDIR}", but I would do the whole thing like:
```suggestion
PATCHES=( "${FILESDIR}/${PN}-1.7.1-fix-LV2_Descriptor.patch"  )
```

Change the ${PN}-1.7.1 to ${P} if this patch is upstreamed.

Comment (RarogCmex) [reply media-sound/yoshimi/yoshimi-1.7.1-r1.ebuild]: It seems like useless labor, but I push it.
